From c3a589334a2512699f0821038f6e158171796a05 Mon Sep 17 00:00:00 2001
From: flawedworld <flawedworld@flawed.world>
Date: Fri, 12 Nov 2021 14:28:42 +0000
Subject: [PATCH 37/37] Add VS 2022 support

---
 build/vs_toolchain.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build/vs_toolchain.py b/build/vs_toolchain.py
index ecf0971ab1349..50f37096b581c 100755
--- a/build/vs_toolchain.py
+++ b/build/vs_toolchain.py
@@ -40,12 +40,14 @@ json_data_file = os.path.join(script_dir, 'win_toolchain.json')
 
 # VS versions are listed in descending order of priority (highest first).
 MSVS_VERSIONS = collections.OrderedDict([
+  ('2022', '17.0'),
   ('2019', '16.0'),
   ('2017', '15.0'),
 ])
 
 # List of preferred VC toolset version based on MSVS
 MSVC_TOOLSET_VERSION = {
+   '2022' : 'VC143',
    '2019' : 'VC142',
    '2017' : 'VC141',
 }
-- 
2.33.0.windows.1

